var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -69,6 +69,21 @@ class PyPyside2(PythonPackage):
     depends_on("libxslt@1.1.19:", when="+doc", type="build")
     depends_on("py-sphinx", when="+doc", type="build")
 
+    def patch(self):
+        filter_file(
+            r"\=\${shiboken_include_dirs}",
+            ':'.join([
+                r"=${shiboken_include_dirs}",
+                self.spec["qt"]["glx"]["libglx"].prefix.include,
+                self.spec["qt"]["libxcb"].prefix.include
+            ]),
+            "sources/pyside2/cmake/Macros/PySideModules.cmake"
+        )
+
+    def setup_build_environment(self, env):
+        if self.spec.satisfies("^llvm@11:"):
```

Comment:
Any reason we shouldn't always use this?

Reply:
Seems to only be a problem from `llvm` 11 onwards, I have updated to just add the envar as it may not matter.